app.rb

```diff
@@ -25,7 +25,7 @@ class Chitter < Sinatra::Base
   end
 
   get '/home' do 
-    @peeps = Peep.all
+    @peeps = Peep.all.reverse
```

Comment:
Suggestion - to use .reverse on the model level, due to pots being displayed always from the latest